work-tracker-spring-boot/README.md

```diff
@@ -216,6 +216,22 @@ private MdcExecutor mdcExecutor;
 //...
 mdcExecutor.execute(someRunnable);
 ```
+
+- **Exclude Urls from Logging**
+
+You can exclude specific request urls from logging from url patterns.
+
+```java
+@Configuration
+public class WorkTrackerConfig extends WorkTrackerConfigurer<SpringWork> {
+    public WorkTrackerConfig() {
+      excludePathPatterns("/health"); // ignores /health*
```

Comment:
Change this to ignoring /facicon.ico and I'll merge it in.  

Generally, I don't want to encourage people to ignore the health endpoints.

Reply:
Fixed